app/controllers/step_by_step_pages_controller.rb

```diff
@@ -199,6 +198,15 @@ def revert_page
     StepByStepPageReverter.new(@step_by_step_page, payload).repopulate_from_publishing_api
   end
 
+  def publish_note_description(publish_intent)
+    if @step_by_step_page.has_been_published?
+      custom_note = " with note: #{publish_intent.change_note}" unless publish_intent.change_note.empty?
+      "#{publish_intent.update_type.capitalize} update published by #{current_user.name}#{custom_note}"
+    else
+      "First published by #{current_user.name}"
+    end
```

Comment:
You could use an `unless` here rather than if/else:

```
return "First published by #{current_user.name}" unless @step_by_step_page.has_been_published?

custom_note = " with note: #{publish_intent.change_note}" unless publish_intent.change_note.empty?
 "#{publish_intent.update_type.capitalize} update published by #{current_user.name}#{custom_note}"
```